tests/integration_tests/scheduler/mock_torque_bin/runner

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+job=$1
+
+function handle_sigterm {
+    # Torque uses (256 + SIGNAL) as the returncode
+    echo "271" > "${job}.returncode"
+    exit 0
+}
+
+trap handle_sigterm SIGTERM
+
+echo "$$" > "${job}.pid"
+sh "${job}.script" > "${job}.stdout" 2> "${job}.stderr"
```

Comment:
The process started here is not killed when this script is killed. I have an extra `kill $child_pid` in `handle_sigterm` and 
```
sh "${job}.script" > "${job}.stdout" 2> "${job}.stderr" &
child_pid=$!

wait
```

Reply:
I didn't read this and ended up doing:
```sh
sh "${job}.script" ... &
wait $!
echo $? > "${job}.returncode"
```